src/azure-cli-core/azure/cli/core/style.py

```diff
@@ -72,3 +72,22 @@ def format_styled_text(styled_text):
     # Reset control sequence
     formatted_parts.append(Fore.RESET)
     return ''.join(formatted_parts)
+
+
+def get_styled_command(raw_command):
+    styled_command = []
+    argument_begins = False
+
+    for index, arg in enumerate(raw_command.split()):
```

Comment:
While `str.split` works here, [`shlex.split`](https://docs.python.org/3/library/shlex.html#shlex.split) is safer which doesn't splits spaces within quoted text, like `"some quoted text"`.

https://github.com/Azure/azure-cli/blob/0c852330023e0746fa4269837204062aed7dc13e/src/azure-cli-testsdk/azure/cli/testsdk/base.py#L289

Reply:
`str.split` works fine here, except for the cases where we have space in the command name. `shlex.split` will remove the quotation marks in the command thus bring more complexity when we join the command args.

Let's use `str.split` here @zhoxing-ms for awareness